src/tracing/errors.test.ts

```diff
@@ -29,4 +29,141 @@ describe('filterKnownErrors', () => {
     const originalException = new SyntaxError("Unexpected token '<'")
     expect(filterKnownErrors(ERROR, { originalException })).toBe(null)
   })
+
+  describe('chunk errors', () => {
+    afterEach(() => {
+      jest.restoreAllMocks()
+    })
+
+    it('filters 499 error coded chunk error', () => {
+      const mockGetEntriesByType = jest.fn(() => {
+        return [
+          {
+            name: 'https://app.uniswap.org/static/js/20.d55382e0.chunk.js',
+            entryType: 'resource',
+            startTime: 253.79999999701977,
+            duration: 3.0999999940395355,
+            initiatorType: 'link',
+            nextHopProtocol: 'h2',
+            renderBlockingStatus: 'non-blocking',
+            workerStart: 254,
+            redirectStart: 0,
+            redirectEnd: 0,
+            fetchStart: 254.19999998807907,
+            domainLookupStart: 254.19999998807907,
+            domainLookupEnd: 254.19999998807907,
+            connectStart: 254.19999998807907,
+            secureConnectionStart: 254.19999998807907,
+            connectEnd: 254.19999998807907,
+            requestStart: 255.09999999403954,
+            responseStart: 256,
+            responseEnd: 256.8999999910593,
+            transferSize: 0,
+            encodedBodySize: 227688,
+            decodedBodySize: 227688,
+            responseStatus: 499,
+            serverTiming: [],
```

Comment:
Only mock what you need:

```suggestion
            name: 'https://app.uniswap.org/static/js/20.d55382e0.chunk.js',
            responseStatus: 499,
```